sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientTest.java

```diff
@@ -331,6 +342,68 @@ void getQueueRuntimePropertiesWithResponse() throws IOException {
             .verifyComplete();
     }
 
+    /* When ServiceBusManagementError is not populated in 'ServiceBusManagementErrorException', we get
+    'ClientAuthenticationException' with error message.*/
+    @Test
+    void getSubscriptionRuntimePropertiesException() {
+        // Arrange
+        final String topicName = "topicName";
+        final String SubscriptionName = "subscriptionName";
```

Comment:
Incorrect casing for local variables.
You also need to format this document in addition to the previous one.

Reply:
Consider using a Parameterised test rather than individual test cases. This is the same code, except the objects are different.